src/test/java/hudson/plugins/copyartifact/ParameterizedBuildSelectorTest.java

```diff
@@ -273,9 +284,13 @@ public void testVariableExpression() throws Exception {
         FreeStyleProject copiee = j.createFreeStyleProject();
         copiee.getBuildersList().add(new FileWriteBuilder("artifact.txt", "foobar"));
         copiee.getPublishersList().add(new ArtifactArchiver("artifact.txt"));
+        // Allow next project to copy from this

```

Comment:
These lines should have been removed with removing `CopyArtifactPermissionProperty`.
You can review your whole changes in the "Files changed" tab in GitHub, and it should be more useful for you than review with `git diff`.